lib/DraggableCore.es6

```diff
@@ -67,6 +67,12 @@ export default class DraggableCore extends React.Component {
     enableUserSelectHack: PropTypes.bool,
 
     /**
+     * `offsetParent`, if set, uses the passed DOM node to compute drag offsets
+     * instead of using the parent node.
+     */
+    offsetParent: PropTypes.bool,
```

Comment:
This isn't a bool. Unfortunately there's no dom node propType so it should be something like:

```js
offsetParent: function(props, propName) {
  if (process.browser && props[propName] && props[propName].nodeType !== 1) {
    throw new Error('Draggable's offsetParent must be a DOM Node.')
  }
}
```